packages/encoding/src/hex.ts

```diff
@@ -1,9 +1,7 @@
 export function toHex(data: Uint8Array): string {
-  let out = "";
-  for (const byte of data) {
-    out += ("0" + byte.toString(16)).slice(-2);
-  }
-  return out;
+  return Array.from(data)
```

Comment:
```suggestion
  return data[Symbol.iterator]()
```

But this only works natively in Node.js 22 and ES2025, so a simple polyfill would be needed.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator/map